ext/reflection/php_reflection.c

```diff
@@ -1379,6 +1404,82 @@ static zend_op *_reflection_param_get_default_precv(INTERNAL_FUNCTION_PARAMETERS
 }
 /* }}} */
 
+/* {{{ _reflection_param_get_default_arg_info */
+static int _reflection_param_get_default_arg_info(zend_internal_arg_info *arg_info, zval *default_value_zval)
```

Comment:
Should be parsing extracted from the 2 places it's currently needed? Maybe into `zend_API` where the `arg_info.default_value` is anyway defined?